cg/cli/workflow/fastq/base.py

```diff
@@ -39,6 +42,17 @@ def store_fastq_analysis(context: click.Context, case_id: str, dry_run: bool = F
         return
     status_db.session.add(new_analysis)
     status_db.session.commit()
+    trailblazer_api: TrailblazerAPI = context.obj.trailblazer_api
+    trailblazer_api.add_pending_analysis(
+        case_id=case_id,
+        analysis_type=AnalysisType.OTHER,
+        data_analysis=Pipeline.FASTQ,
+        config_path="",
+        out_dir="",
+        slurm_quality_of_service=Priority.priority_to_slurm_qos().get(case_obj.priority),
```

Comment:
If these should be changed, please let me know. As far as we could tell, there was no need to set them, given that no actual analysis is run for a Fastq case.

Reply:
Out of scope.